### DISCUSSION TITLE
Issue with syntax highlighting

### DISCUSSION THREAD

**Discussion by abduraufsherkulov** [ben-rogerson/twin.macro#564] [Q&A]
![image](https://user-images.githubusercontent.com/17489128/138249770-33a920ab-0293-49ae-a615-e25862714b12.png)
Syntax highlighting is not working as expected, after `tw` it is plain white. Instead I want dark orange type of color just like in `className`'s property. Is there any suggestion how to fix that?

**Comment by ben-rogerson** [comment]
Check out the [Tailwind Twin IntelliSense docs](https://marketplace.visualstudio.com/items?itemName=lightyen.tailwindcss-intellisense-twin) - theres some config in that package for colors.
